charts/rancher-eks-operator/1.1.100/values.yaml

```diff
@@ -4,8 +4,9 @@ global:
 eksOperator:
   image:
     repository: rancher/eks-operator
-    tag: v1.0.8-rc2
+    tag: v1.0.8
```

Comment:
We have rc for 1.1.1 already, should be updated in app-version too. We should really tag a new RC, there are newer commits now.

Reply:
Done